packages/amplify_flutter/test/config/testdata/auth/old/auth_with_email.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
What is the significance of the folder name "old" here? Is this an output from an older CLI version? Do we want to name to folder something like `cli-6.1.0` to clarify what version we are testing against? The same goes for "zero_config"

Reply:
I suppose - zero config is a CLI milestone, old means before that

Reply:
What CLI version is zero config? What is the oldest version we aim to support? If these tests need to be updated, how would someone know what version of the CLI to use to regenerate the configs?

"zero config" is sort of a buzz word we are using now, but in a year from now will someone know what "old" and "zero config" means?